CROS_RUST_REMOVE_DEV_DEPS didn't cover all the dev dependencies so this patch
was needed.
diff --git a/Cargo.toml b/Cargo.toml
index 7a7f733..ea28465 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -51,20 +51,3 @@ version = "0.2.57"
 
 [dependencies.void]
 version = "1.0.2"
-[dev-dependencies.bytes]
-version = "0.4.8"
-
-[dev-dependencies.lazy_static]
-version = "1.2"
-
-[dev-dependencies.rand]
-version = ">= 0.6, < 0.7"
-
-[dev-dependencies.tempfile]
-version = "3.0.5"
-[target."cfg(any(target_os = \"android\", target_os = \"linux\"))".dev-dependencies.caps]
-version = "0.3.1"
-[target."cfg(target_os = \"dragonfly\")".build-dependencies.cc]
-version = "1"
-[target."cfg(target_os = \"freebsd\")".dev-dependencies.sysctl]
-version = "0.1"
